src/main/java/Buu/Buu.java

```diff
@@ -0,0 +1,68 @@
+package Buu;
+
+import java.util.List;
+
+import javafx.application.Platform;
+
+
+/**
+ * The main class for the GPT application, which handles the initialization of the
+ * user interface, storage, and task list, and runs the main application loop.
+ */
+public class Buu {
+    private Ui ui;
+    private Storage storage;
+    private TaskList taskList;
+
+    /**
+     * Constructs a new GPT application with a default file path for task storage.
+     */
+    public Buu() {
+        String filePath = "data/tasks.txt"; // Default file path
+        ui = new Ui();
+        storage = new Storage(filePath);
+        taskList = new TaskList(storage.loadTasks());
+        // Show the welcome message when the Buu application starts
+        ui.showWelcomeMessage("Buu"); // This adds the welcome message
```

Comment:
It is generally not good practice for a constructor to have side-effects such as displaying UI elements. It might be better to move this down to an initialisation method such as in main() itself.